src/components/productDetail.js

```diff
@@ -26,7 +26,7 @@ function ProductDetail({ productInformation }) {
         </div>
         <h2 className="product-title">{productInformation.name}</h2>
         <div className="star-section">
-          <p>★★★★☆</p>
+          <p>{productInformation.ratingAverage}</p>
```

Comment:
Here are we just showing the number?
The original idea is to show the same starts than the rating average, where if the product has a 4 rating average, it should show 4 filled stars and just one not filled, and so on.